### PR TITLE
Create IMEI Fraud API

### PR DESCRIPTION
**Pull request by wwAMRA**
IMEI Fraud API is part of Camara drop 2 submission.

### CODE REVIEW

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud [hunk 1]
```diff
@@ -0,0 +1,14 @@
+IME Fraud API submission
```
Comment (chrishowell) [inline APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:1]:
```suggestion
IMEI Fraud API submission
```

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+| **Field** | Description |
+| ---- | ----- |
+| API name | IMEI Fraud|
+| API owner | MTN - Waseem Amra |
+| Initial API Contributors | MTN - Waseem Amra | 
+| API summary | This API allows for the API Consumer to understand if fraud has been associated to the device. Fraud can range from lost, suspended, stolen, blacklisted. |
```
Comment (chrishowell) [inline APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:8]:
```suggestion
| API summary | This API allows for the API Consumer to understand if any fraud markers have been associated with the device. Fraud markers may be applied if the device is lost, suspended, stolen, blacklisted, etc. |
```

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud logs are associated with the device. </em>.
```
Comment (chrishowell) [inline APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:9]:
```suggestion
| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud markers are associated with the device. </em>.
```

**Comment by jordonezlucena** [comment]
@wwAMRA: please review the suggestions and accept them, if they are acceptable for you

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud (hunk 2 above)
Comment (wwAMRA) [reply APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:8]: Accepted

**Review by wwAMRA — COMMENTED** [review]
Approved

**Comment by jordonezlucena** [comment]
This API complies with the template.

**Review by ShutingQing — DISMISSED** [review]
LGTM

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud (hunk 1 above)
Comment (hdamker) [reply APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:1]: @wwAMRA can you please commit this correction into the PR?

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud (hunk 2 above)
Comment (hdamker) [reply APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:8]: @wwAMRA if you are ok with this proposed change, can you please commit it into the PR?

File: APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud (hunk 3 above)
Comment (hdamker) [reply APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud:9]: @wwAMRA if would expect that you either commit the change or comment why the change is not appropriate

**Review by hdamker — DISMISSED** [review]
@wwAMRA @jordonezlucena the "approved" proposed changes need to be committed into the file before a merge

**Review by eric-murray — APPROVED** [review]

**Pull request merged**